hack/boilerplate/kubernetes/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright YEAR The KCP Authors.
```

Comment:
Shouldn't this say "The Kubernetes Authors"?

Reply:
It should. Why does it work?

Reply:
> checked 0 files, ignored 2.